doc/specs/#5000 - Process Model 2.0/#5000 - Process Model 2.0.md

```diff
@@ -967,7 +967,7 @@ of each other.
    - Summon the nearest window
    - make the window "drop down" from the top
    - Summon the MRU window
-     - It would need to track a the MRU for windows, so pressing the shortcut when
+     - It would need to track the MRU for windows, so pressing the shortcut when

```

Comment:
This is another forbidden pattern (two articles in a row). Corrections to this require a lot more brain power than I've had of late. I could easily be wrong for each instance...